modules/rosa-create-objects.adoc

```diff
@@ -273,6 +273,81 @@ Create a cluster with autoscaling enabled on the default worker machine pool.
 $ rosa create cluster --cluster-name=mycluster -region=us-east-1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
 
+[id="rosa-create-external-auth-provider_{context}"]
+== create external-auth-provider
+
+Add an external authentication provider instead of an internal OIDC provider to approve users to access your cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create external-auth-provider --cluster=<cluster_name> | <cluster_id> [arguments]
+----
```

Comment:
It would be better to add note to highlight it is only supported for HCP cluster that uses external authentication providers.